src/Common/ThreadFuzzer.cpp

```diff
@@ -146,7 +153,7 @@ bool ThreadFuzzer::isEffective() const
             return true; \
         if (NAME##_before_sleep_probability.load(std::memory_order_relaxed) > 0.0) \
             return true; \
-        if (NAME##_before_sleep_time_us_max.load(std::memory_order_relaxed) > 0.0) \
+        if (NAME##_before_sleep_time_us_max.load(std::memory_order_relaxed) > 0.001) \
```

Comment:
This looks strange to me.

Reply:
I'll change it to check it in the `sleepForNanoseconds` call instead. The problem comes from trying to cast to an integer and calculate a mod % value